code/modules/antagonists/vampire/thrall.dm

```diff
@@ -0,0 +1,57 @@
+/datum/antagonist/subordinate/thrall
+	id = ROLE_VAMPTHRALL
+	display_name = "vampire thrall"
+	remove_on_death = TRUE
```

Comment:
what about SR and vampire blood revivals?

Reply:
Both SR and vampire blood revivals have the prerequisite of the thrall being dead, meaning the antagonist status will have already been removed.

Reply:
Ahh okay, that makes sense, assuming the thrall revive process re-adds the antagonist status?

Reply:
Indeed, yes.